app/helpers/flash_helper.rb

```diff
@@ -1,6 +1,6 @@
 module FlashHelper
   # Flash type -> Bootstrap alert class
   def flash_mapping
-    { notice: :info, error: :danger, success: :success }
+    { notice: :info, error: :danger, authorization_error: :danger, success: :success }
```

Comment:
What would happen if, then, we needed the warning for a more generic "warning" message?
Because, as far as I understood, the reason why @msufa introduced a new custom error type is because he needs to check if the error was caused by an `:authorization_error` when rendering the flash message in the view, to decide wether the link to request access should be displayed or not (see `_alerts.html.erb` below).

So, I'm curious here: is there another way, to keep the default error types provided by Rails and display the link as part of the error message only when needed, according to the conditions given by the `display_access_request_link?` helper?

The only alternative I found would be something in the likes of: http://stackoverflow.com/questions/26538891/flash-message-with-html-safe-from-the-controller-in-rails-4-safe-version